src/main/java/org/json/XML.java

```diff
@@ -853,8 +870,15 @@ private static String toString(final Object object, final String tagName, final
                     sb.append(indent(indent));
                     sb.append('<');
                     sb.append(key);
-                    sb.append("/>");
-                    if(indentFactor > 0){
+                    if (!keepSelfClosingTags) {
```

Comment:
`if (!condition) code1 else code2`

can be replaces with

`if (condition) code2 else code1`

Reply:
Yeah, sure. I'll change it. I am thinking to change my approach though.